src/cpp/proxy/domainmap.cpp

```diff
@@ -709,6 +708,8 @@ class DomainMap::Thread : public QThread
 {
 	Q_OBJECT
 
+	Connection startedConnection;
```

Comment:
The connection only needs to exist within `run()` so it doesn't need to be a class member.